### PR TITLE
Update security-hardening-for-github-actions.md

### PR DESCRIPTION
**Pull request by fuxingloh**
### Why:

Confusing statement, shouldn't it be {% data variables.product.prodname_actions %} instead of {% data variables.product.product_name %}?

### What's being changed:

https://docs-3706--patch-1.herokuapp.com/en/actions/learn-github-actions/security-hardening-for-github-actions#considering-cross-repository-access

**Before:** GitHub is intentionally scoped for a single repository at a time. The GITHUB_TOKEN grants the same level of access as a write-access user, because any write-access user can access this token by creating or modifying workflow files.

**After:** GitHub Actions is intentionally scoped for a single repository at a time. The GITHUB_TOKEN grants the same level of access as a write-access user, because any write-access user can access this token by creating or modifying workflow files.

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by Ala619** [comment]
1a940d541f252f59d05bf83afbd240bc0f9c45e4

**Comment by kkash18** [comment]
> ### Why:
> Confusing statement, shouldn't it be {% data variables.product.prodname_actions %} instead of {% data variables.product.product_name %}?
> 
> ### What's being changed:
> https://docs-3706--patch-1.herokuapp.com/en/actions/learn-github-actions/security-hardening-for-github-actions#considering-cross-repository-access
> 
> **Before:** GitHub is intentionally scoped for a single repository at a time. The GITHUB_TOKEN grants the same level of access as a write-access user, because any write-access user can access this token by creating or modifying workflow files.
> 
> **After:** GitHub Actions is intentionally scoped for a single repository at a time. The GITHUB_TOKEN grants the same level of access as a write-access user, because any write-access user can access this token by creating or modifying workflow files.
> 
> ### Check off the following:
> * [x]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [ ]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Review by kkash18 — COMMENTED** [review]
`good `sho``

**Review by kkash18 — APPROVED** [review]

File: content/actions/learn-github-actions/security-hardening-for-github-actions.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ This means that a compromise of a single action within a workflow can be very si
 
 ### Considering cross-repository access
```
Comment (kkash18) [inline content/actions/learn-github-actions/security-hardening-for-github-actions.md:77]:
```sugg![1f3ad.png](https://user-images.githubusercontent.com/77268170/107938858-92241280-6f4b-11eb-907c-2dc30c6336c8.png)
```

Comment (kkash18) [reply content/actions/learn-github-actions/security-hardening-for-github-actions.md:77]:
```

**Comment by janiceilene** [comment]
@fuxingloh Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Review by shati-patel — APPROVED** [review]
Thank you for fixing this, @fuxingloh! Much appreciated 😃 

I'll get this merged shortly.

**Comment by shati-patel** [comment]
We're looking at the Browser Test failure internally, we can ignore it for now!

**Pull request merged**